bucket/squirreldisk.json

```diff
@@ -0,0 +1,34 @@
+{
+    "version": "0.3.4",
+    "homepage": "https://www.squirreldisk.com/",
+    "description": "A beautiful, super fast disk usage analysis tool. Built with rust.",
+    "license": "AGPL-3.0",
+    "notes": [
+        "This app depends on MSEdgeWebview2 and VB-Cable Virtual Audio Device to function properly.",

```

Comment:
A dependency to VB-Cable ? I can't find any reference to that on their repository.